clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -213,7 +213,7 @@
  * <h4>Manual Offset Control</h4>
  *
  * Instead of relying on the consumer to periodically commit consumed offsets, users can also control when records
- * should be considered as consumed and hence commit their offsets. This is useful when the consumption of the
+ * should be considered as consumed and hence commit their offsets. This is useful when the consumption of the messages
```

Comment:
@enothereska is right, if you make that additional change, I'll merge your PR. Thanks!